sample-app/src/main/java/com/tink/link/app/navToFlows/FlowCases.kt

```diff
@@ -709,7 +710,8 @@ fun showAccountAggregationRefreshCredentials(
     val refreshCredentials = RefreshCredentials(
         authorizationCode = NecessaryIds.authorizationCode,
         credentialsId = NecessaryIds.credentialsId,
-        market = "SE"
+        market = "SE",
+        locale = Locale("sv", "SE")
```

Comment:
Do we need to have market at 2 places?

Reply:
But its only one market and one locale 🤔